android-components/components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketStoriesConfig.kt

```diff
@@ -38,8 +40,16 @@ class PocketStoriesConfig(
         DEFAULT_SPONSORED_STORIES_REFRESH_INTERVAL,
         DEFAULT_SPONSORED_STORIES_REFRESH_TIMEUNIT,
     ),
+    val sponsoredStoriesParams: PocketStoriesRequestConfig = PocketStoriesRequestConfig(),
 )
 
+/**
+ * Configuration for sponsored stories request indicating parameters used to set spoc content.
+ *
+ * @property siteId ID of the site parameter.
```

Comment:
Could we offer more context for the future adding that based on this the app will receive a different collection of sponsored stories?